src/components/header/template.njk

```diff
@@ -56,38 +55,29 @@
         {% endif %}
       </a>
     </div>
-
-    {% if params.serviceName or params.navigation or params.signOutHref or params.languageToggle %}
+    {% if params.serviceName or params.navigation  %}
```

Comment:
Isn't the `params.signOutHref` and `params.languageToggle` on of the reasons this was forked initially? Should we keep these in?

Reply:
+1 What's the reason for taking these out?